openfisca_france/model/prestations/aide_mobilite.py

```diff
@@ -0,0 +1,377 @@
+                                                    + ((distance_aller_retour > parametres_amob.distance_minimum.hors_metropole) * (not_(reside_en_metropole) * residence_renseignee))
+                                                    + ((temps_de_trajet > temps_de_trajet_min) * residence_renseignee))
+
+        #  7
+        dispositifs_formations = individu('dispositifs_formation', period)
+        dispositifs_formations_eligibles = dispositifs_formations == DispositifsDeFormation.autre
+
+        eligibilite_amob = (contextes_eligibles
+                       * activites_eligibles
+                       * categories_eligibles
+                       * montants_allocations_eligibles
+                       * lieux_activite_eligibles
+                       * distances_et_durees_aller_retour_eligibles
+                       * dispositifs_formations_eligibles)
+
+        return eligibilite_amob
+
+
+class aide_mobilite(Variable):
+    value_type = float
+    entity = Individu
+    label = "Calcul du montant de l'aide à la mobilité - AMOB"
+    definition_period = MONTH
+    set_input = set_input_divide_by_period
+    reference = "http://www.bo-pole-emploi.org/bulletinsofficiels/deliberation-n-2021-42-du-8-juin-2021-bope-n2021-43.html?type=dossiers/2021/bope-n-2021-043-du-11-juin-2021"
+
+    def formula_2021_06_09(individu, period, parameters):
+
+        eligibilite_amob = individu('aide_mobilite_eligible', period)
+        parametres_amob = parameters(period).prestations_sociales.aide_mobilite
+
+        annee_glissante = period.start.period('year').offset(-1)
+
+        aide_mobilite_12_derniers_mois = individu('aide_mobilite', annee_glissante, options=[ADD])
+
+        montant_max = parametres_amob.montants.maximum
+        montant_amob_deja_percu = min_(montant_max, fabs(aide_mobilite_12_derniers_mois))
+        distance_aller_retour = individu('distance_aller_retour_activite_domicile', period)
+        nb_aller_retour = individu('nombre_allers_retours', period)
+        nb_nuitees = individu('nuitees', period)
+        nb_repas = individu('repas', period)
+
+        montant = parametres_amob.montants
+
+        montants_frais_deplacement = montant.deplacement * distance_aller_retour * nb_aller_retour
+        montants_frais_hebergement = montant.hebergement * nb_nuitees
+        montants_frais_repas = montant.repas * nb_repas
+
+        montants_theoriques = montants_frais_deplacement + montants_frais_hebergement + montants_frais_repas
+        montants_max_attribuables = montant_max - montant_amob_deja_percu
+
+        montants_reels = min_(montants_theoriques, montants_max_attribuables)
+
+        return montants_reels * eligibilite_amob
+9
+
+class aide_mobilite_bon_de_transport(Variable):
+    value_type = bool
+    entity = Individu
+    label = "Attribution d'un bon de transport dans un contexte précis - AMOB"
```

Comment:
Comment interpréter `dans un contexte précis` en tant que définition ? 🤔 Proposition :
```suggestion
    label = "Eligibilité à un bon de transport, part de l'aide à la mobilité de Pôle emploi (AMOB)"
```